Adding Drell-Yan operators to rge database

TommasoArmadillo opened the pull request:
This PR adds all the 2 leptons 2 quarks operators that appear in the Drell-Yan dataset to the rge dictionary. This follows PR #128.

Namely the operators are:

$O_{eu}$, $O_{ed}$, $O_{eb}$, 
$O_{\ell u}$, $O_{\ell d}$, $O_{\ell b}$, 
$O_{\ell q}^{(-)}$, $O_{\ell q}^{(3)}$,
$O_{qe}$.

LucaMantani commented on src/smefit/rge/wcxf.py:
```diff
@@ -171,10 +180,22 @@
-    "OQe": {"wc": ["qe_3311"]},
-    "OQl31": {"wc": ["lq3_1133"]},
-    "OQlM1": {"wc": ["lq1_1133", "lq3_1133"], "coeff": [1.0, -1.0]},
+    # 2 quark 2 lepton operators
+    "Oeu": {"wc": ["eu_1111", "eu_1122"]},
+    "Oed": {"wc": ["ed_1111", "ed_1122"]},
+    "Oeb": {"wc": ["ed_1133"]},
     "Otl1": {"wc": ["lu_1133"]},
     "Ote": {"wc": ["eu_1133"]},
+    "Olq31": {"wc": ["lq3_1111", "lq3_1122"]},
+    "OlqM1": {
+        "wc": ["lq1_1111", "lq1_1122", "lq3_1111", "lq3_1122"],
+        "coeff": [1.0, 1.0, -1.0, -1.0],
+    },
+    "OQl31": {"wc": ["lq3_1133"]},
+    "OQlM1": {"wc": ["lq1_1133", "lq3_1133"], "coeff": [1.0, -1.0]},
+    "Olu": {"wc": ["lu_1111", "lu_1122"]},
+    "Old": {"wc": ["ld_1111", "ld_1122"]},
+    "Olb": {"wc": ["ld_1133"]},
+    "Oqe": {"wc": ["qe_1111", "qe_2211"]},
+    "OQe": {"wc": ["qe_3311"]},
```
I am not sure the ones with multiple indices are correct here. Take Oeu.

When you turn it on, you indeed turn on eu_1111 and eu_1122. Now here you are implementing the inverse transformation. Imagine you get the following result:

{eu_1111: 2.0, eu_1122: 2.0} which means that Oeu = 2.0, but the way you wrote it there instead would sum the two contributions and give you Oeu = 4.0.

Now, if you see above how I did in similar cases, I decided to read only the component 1. The underlying assumption is that the number 2 will be identical, ie. that we have not broken the flavour symmetries... This is clearly something that can be potentially problematic but at the time I thought it was the simplest way (and I checked that flav symmetries were respected in simple tests).

LucaMantani reviewed (changes requested):
I know why you called Olq31 and OlqM1 that way, and it makes sense, but at the moment they are in contrast with the version with big Q... I think it's better if we uniform them?

TommasoArmadillo commented:
Hi Luca,

I agree with the comment on the names of the operators, we should uniform them and try to use a standard notation. I renamed `OlqM1 -> OqlM1` and `Olq31 -> Oql31` to match the one of big Q.

Concerning the inverse transformation one, I see why it is wrong written in this way. I corrected them by reading only the first component, with the assumption that all the others are the same. For all the operators involved I explicitly checked that if I set `yukawa: none` or `yukawa: top` in the rge evolution options, then the wc are the same (e.g. I always find eu_1111= eu_1122). I also checked that if `yukawa:  none` this statement is not true anymore, which is what we would expect.

LucaMantani reviewed (approved):
Looks good!
